src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        String [] data = new String[] {
```

Comment:
The array `data` should have a more informative name that reflects its purpose, like `salaryData` or `employeeRecords`.